src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -107,6 +107,9 @@ public class Startup(IConfiguration configuration, IHostEnvironment hostEnvironm
         // This method gets called by the runtime. Use this method to add services to the container.
         public virtual void ConfigureServices(IServiceCollection services)
         {
+            // TODO EES- Remove this when the Public Data db exists in ALL Azure environments.
+            var publicDataDbExists = configuration.GetValue<bool>("PublicDataDbExists");
```

Comment:
Does this just need adding to appsettings.Development.json?

Reply:
I debugged this when running locally and it defaults to false when not set, rather than throwing an exception.

I think that makes sense because looking at the source code for `GetValue<bool>` it is `return GetValue(configuration, key, default(T));`

Reply:
Added it in now so that we configure `DataSetService` in the local environment like this.

```csharp
services.AddTransient<IDataSetService, DataSetService>();
```